plaso/storage/sqlite/sqlite_file.py

```diff
@@ -255,7 +253,7 @@ def _GetAttributeContainerByIdentifier(self, container_type, identifier):
       IOError: if an invalid identifier is provided.
```

Comment:
please update docstring for new raise condiditon

Reply:
Done